website/tests/test-helper.js

```diff
@@ -4,9 +4,18 @@ import * as QUnit from 'qunit';
 import { setApplication } from '@ember/test-helpers';
 import { setup } from 'qunit-dom';
 import { start } from 'ember-qunit';
+import { setRunOptions } from 'ember-a11y-testing/test-support';
 
 setApplication(Application.create(config.APP));
 
+// TODO why isn't this working?
```

Comment:
This _should_ work, but it doesn't.  Maybe because it's a non-standard prember app? (either of those things would make it challenging)

Reply:
Is there an error your seeing? Even though we are using prember I don't think there is anything non-standard about how the tests are being run.

Reply:
There's not an error, there's just...nothing. I tried running it with `PREMBER=true ember s -p 0" and then visiting /tests in my browser.

Reply:
🤔  That seems correct since tests shouldn't be part of a production build. Any reason not to `ember s` and then visit `/tests`?

Reply:
oh! I should have said, I tried that _first_. Then I thought, maybe I need to `PREMBER=true` first. So neither worked for me. 🤔